integration/tilt_args_test.go

```diff
@@ -22,8 +23,19 @@ func TestTiltArgs(t *testing.T) {
 
 	f.logs.Reset()
 
+	// Just run
```

Comment:
not sure what this means - is it cruft?

Reply:
maybe it was trying to be motivational? :joy: